build.sh

```diff
@@ -71,7 +82,18 @@ else
 	popd
 fi
 
-if [ "$ARCH" = "arm" ]; then
+if [[ $TARGET_OS == 'android' ]]; then
+	echo "Setting gclient target_os to android"
+	echo "target_os = [ 'android' ]" >> $WEBRTC_DIR/.gclient
```

Comment:
This is going to make $WEBRTC_DIR/.gclient longer by a line every time build.sh is run. Maybe do something like this (untested):
```suggestion
	grep -q -E '^\s*target_os\s*=.*\<android\>' $WEBRTC_DIR/.gclient || echo "target_os = [ 'android' ]" >> $WEBRTC_DIR/.gclient
```
The above still has the problem that if someone has manuall configured e.g. `target_os = [ 'win' ]`, then the command will stomp over it rather than rewriting to `target_os = [ 'win', 'android' ]`... But maybe we can ask people to handle cases like that manually.

Reply:
I couldn't get the _gclient_ stuff to run repeatedly.  What is included there are the steps that I took manually, based on the Chromium docs, to get things building in a fresh VM.  I've never used _gclient_ before, so I'm bailing out now, waiting until someone who does know it can do the right then.  I don't think tweaking that stuff is worth doing until then.

Reply:
In my branch I came up with some sed magic to manipulate the `target_os` variable.
```
sed -i "/^target_os *= *.*\\<android\\>/{p;h;d}; /^target_os *= */{s/ *]/, 'android'&/;p;h;d}; \${x;s/^target_os/&/;tx;atarget_os = [ 'android' ]"$'\n'";:x x}" $WEBRTC_DIR/.gclient
```
1. If there's a `target_os` line and `android` is already present, leave it alone.
2. If there's a `target_os` line and `android` is missing, append `android`.
3. If there's no `target_os` line, add one.